federation/v2.0/federation-v2.0.graphql

```diff
@@ -0,0 +1,13 @@
+directive @key(fields: FieldSet!, resolvable: Boolean = true) repeatable on OBJECT | INTERFACE
+directive @requires(fields: FieldSet!) on FIELD_DEFINITION
+directive @provides(fields: FieldSet!) on FIELD_DEFINITION
+directive @external on OBJECT | FIELD_DEFINITION
+directive @shareable on FIELD_DEFINITION | OBJECT
+directive @extends on OBJECT | INTERFACE
+directive @override(from: String!) on FIELD_DEFINITION
+directive @inaccessible on
```

Comment:
In the implem, `federation/v2.0` essentially gives you `inaccessible/v0.2` and `tag/v0.2`. It seems to be missing the `@tag` definition and the `@inaccessible` locations should match that of v0.2.

Reply:
done